src/internal/components/chart-series-details/styles.scss

```diff
@@ -7,10 +7,54 @@
 @use '../../styles/tokens' as awsui;
 @use './motion';
 
+@use '../chart-series-marker/styles' as marker;
+
+@mixin bold {
```

Comment:
Do we really need this mixin? It is just one line of code using twice. If it is important to secure the value is in sync a variable can do.

Reply:
Replaced by a variable [here](https://github.com/cloudscape-design/components/pull/1644/commits/2bd0f79899478e2344440aceb9589caa92aaddd5)